cmd/rule-evaluator/main.go

```diff
@@ -855,59 +901,52 @@ func (db *queryAccess) Close() error {
 
 // makeInstrumentedRoundTripper instruments the original RoundTripper with middleware to observe the request result.
 // The new RoundTripper counts the number of query requests sent to GCM and measures the latency of each request.
-func makeInstrumentedRoundTripper(transport http.RoundTripper, reg prometheus.Registerer) http.RoundTripper {
-	queryCounter := prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "rule_evaluator_query_requests_total",
-			Help: "A counter for query requests sent to GCM.",
-		},
-		[]string{"code", "method"},
-	)
-	queryHistogram := prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
-			Name:    "rule_evaluator_query_requests_latency_seconds",
-			Help:    "Histogram of response latency of query requests sent to GCM.",
-			Buckets: prometheus.DefBuckets,
-		},
-		[]string{"code", "method"},
-	)
-	reg.MustRegister(queryCounter, queryHistogram)
-
+func makeInstrumentedRoundTripper(transport http.RoundTripper) http.RoundTripper {
 	return promhttp.InstrumentRoundTripperCounter(queryCounter,
 		promhttp.InstrumentRoundTripperDuration(queryHistogram, transport))
 }
 
 type ruleEvaluator struct {
-	queryFunc    rules.QueryFunc
-	rulesManager *rules.Manager
+	queryFunc         rules.QueryFunc
+	rulesManager      *rules.Manager
+	lastEvaluatorOpts *evaluatorOptions
+}
+
+// Returns the URL that points to the rule-evaluator instance. By default, or if using a Google
```

Comment:
How this URL points to rule-eval instance? I don't see any logic for this in this function 🤔

Reply:
The user would set it to the rule-eval instance. I clarified the comment. Thanks!